Yasgui/packages/yasqe/src/sparql.ts

```diff
@@ -92,6 +108,32 @@ export async function executeQuery(yasqe: Yasqe, config?: YasqeAjaxConfig): Prom
   }
 }
 
+export function executeCountQuery(yasqe: Yasqe, config?: YasqeAjaxConfig): void {
+  let req: superagent.SuperAgentRequest;
+  const populatedConfig = getAjaxConfig(yasqe, config);
+  if (!populatedConfig) {
+    //nothing to query
+    return;
+  }
+  if (populatedConfig.reqMethod === "POST") {
+    req = superagent.post(populatedConfig.url).type("form").send(populatedConfig.args);
+  } else {
+    req = superagent.get(populatedConfig.url).query(populatedConfig.args);
+  }
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  req.accept(populatedConfig.accept).set(populatedConfig.headers || {});
+
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  if (populatedConfig.withCredentials) req.withCredentials();
+
+  yasqe.emit("countQuery", req, populatedConfig);
+  req.then((countResponse) => {
```

Comment:
This is still bugging me. We don't handle the errors here, neither in the method client. What would happen if the request fails with some error or timeouts?!